apps/emqx/src/emqx_misc.erl

```diff
@@ -557,16 +557,47 @@ try_to_existing_atom(Convert, Data, Encoding) ->
         _:Reason -> {error, Reason}
     end.
 
-is_sensitive_key(token) -> true;
-is_sensitive_key("token") -> true;
-is_sensitive_key(<<"token">>) -> true;
-is_sensitive_key(password) -> true;
-is_sensitive_key("password") -> true;
-is_sensitive_key(<<"password">>) -> true;
-is_sensitive_key(secret) -> true;
-is_sensitive_key("secret") -> true;
-is_sensitive_key(<<"secret">>) -> true;
-is_sensitive_key(_) -> false.
+is_sensitive_key(token) ->
+    true;
+is_sensitive_key("token") ->
+    true;
+is_sensitive_key(<<"token">>) ->
+    true;
+is_sensitive_key(password) ->
+    true;
+is_sensitive_key("password") ->
+    true;
+is_sensitive_key(<<"password">>) ->
+    true;
+is_sensitive_key(secret) ->
+    true;
+is_sensitive_key("secret") ->
+    true;
+is_sensitive_key(<<"secret">>) ->
+    true;
+is_sensitive_key([{str, Bin}]) when is_binary(Bin) ->
+    is_sensitive_key(Bin);
+is_sensitive_key(Atom) when is_atom(Atom) ->
+    is_sensitive_key(erlang:atom_to_binary(Atom));
+is_sensitive_key(Bin) when is_binary(Bin), (size(Bin) =:= 19 orelse size(Bin) =:= 13) ->
```

Comment:
Add to unit tests further down the module?

Reply:
Localized the redacting as you suggested so this module is not the right place anymore but maybe it is a good idea to have unit tests anyway. I will try to add something similar to what @olcai did  here: https://github.com/emqx/emqx/pull/9840/files#diff-9c8db1387f528a80196a9d0b371694e51d55ebb27bfa8f960abefc1ed05634c6R629

Reply:
Added a commit with a unit test case as well. Not sure if such test cases are executed automatically in the CI runs or if one have to add it manually. Do you happen to know?